Settings-test.toml

```diff
@@ -1,5 +1,10 @@
 [postgres]
-config_string = "postgresql://geoengine:geoengine@localhost:5432"
+host = "localhost"
+port = 5432
+database = "geoengine"
+schema = "pg_temp" # we need the right to create new schemata for tests
```

Comment:
where is this schema used? In the test you create a geoengine_test_1234 schema.

Reply:
I wanted to specify a schema that doesn't provide any harm as default. You don't want to override your public schema on false usage.